fabfile.py

```diff
@@ -1544,6 +1545,19 @@ def install_java():
 
 
 @task
+def install_yarn():
+    """Install yarn"""
+    if not env.mac:
+        if not exists('/etc/apt/sources.list.d/yarn.list'):
```

Comment:
All looks well. One question. If one of our production servers are on Ubuntu 14.04, will this step also be required? 
https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions